packages/mui-material/package.json

```diff
@@ -103,6 +103,11 @@
   "engines": {
     "node": ">=14.0.0"
   },
+  "exports": {
+    "./ButtonBase/TouchRipple": {
+      "mui-src": "./src/ButtonBase/TouchRipple.ts"
+    }
+  },
```

Comment:
What's this export used for?

Reply:
Backwards compatibility for X, it's imported there. We likely want to support this export for one of their majors, until fully removed.